tools/c7n_azure/c7n_azure/tags.py

```diff
@@ -74,7 +75,7 @@ def add_tags(tag_action, resource, tags_to_add):
 
         # add or update tags
         for key in tags_to_add:
-
+            tags_to_add[key] = Lookup.extract(tags_to_add[key], resource)

```

Comment:
Can you please add a comment to explain this? Took me some time to realize `tags_to_add[key]` can be a dict now..